libbeat/docs/newbeat.asciidoc

```diff
@@ -92,49 +99,92 @@ The following example shows an event object in Lsbeat:
 
 Now that you have the big picture, let's dig into the code.
 
-=== Generate your Beat
+=== Generating Your Beat
+
+To generate your own Beat, you use the Beat generator available in the beats repo on GitHub. If you haven't
+downloaded the Beats source code yet, follow the instructions in <<newbeat-getting-ready>>.
 
-To generate your own beat we use the `beat-generator`. First you must install
-https://github.com/audreyr/cookiecutter[cookiecutter]. Check out the
-http://cookiecutter.readthedocs.io/en/latest/installation.html[installation
-guide]. After having installed cookiecutter, we must decide on a name for our beat. 
-The name must be one word all lowercase. In our example is Testbeat.
+Before running the Beat generator, you must install https://github.com/audreyr/cookiecutter[cookiecutter], a
+command-line utility that creates projects from project templates. Check out the 
+http://cookiecutter.readthedocs.io/en/latest/installation.html[installation guide]. After installing cookiecutter,
+decide on a name for your beat. The name should be one word with the first letter capitalized. In our example,
+we use `Testbeat`.
```

Comment:
@monicasarbu Are you OK with me changing Testbeat to Countbeat? I agree with this change because it would be better for us to provide an example that's in the spirit of how we want Beats to be named.